server/datastore/mysql/software_installers.go

```diff
@@ -2,14 +2,76 @@ package mysql
 
 import (
 	"context"
-	"database/sql"
+ 	"database/sql"
 
 	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
 	"github.com/fleetdm/fleet/v4/server/fleet"
 	"github.com/google/uuid"
 	"github.com/jmoiron/sqlx"
 )
 
+func (ds *Datastore) ListPendingSoftwareInstallDetails(ctx context.Context, hostID uint) ([]*fleet.SoftwareInstallDetails, error) {
```

Comment:
is the intention to use this method in the orbit config endpoint later on? if so I think we only need to select the `execution_id`, and we don't need any joins.